src/output.cpp

```diff
@@ -713,6 +713,16 @@ int query_int( const char *mes, ... )
     return atoi( raw_input.c_str() );
 }
 
+std::string query_string( const char *mes, ... )
```

Comment:
Alternatively refactor `bool query_int( std::string msg, bool &result )` where the return is whether the value was set. There aren't many uses to update and almost all are within the debug menu code.